config/locales/en.yml

```diff
@@ -57,30 +63,30 @@ en:
   address: 'Address'
   city: 'City'
   state: 'State'
+  zip_where_can_drive: 'Zip where you can drive'
```

Comment:
"Zip code in swing state where you can drive" ?

If we want to support going to a nearby state to help out, people may not know the zip code. Perhaps we can add a  separate drop down (and column) for drivers to specify "I'm willing to drive in this swing state:"?